Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -216,8 +222,8 @@ function HasElements
 #
 function IsPrBuild
 {    
-    $sourceBranch = $env:Build_SourceBranch
-    $scProvider = $env:Build_Repository_Provider
+    $sourceBranch = GetTaskContextVariable "Build.SourceBranch"
```

Comment:
All calls to task variables or build variables should be made through the helpers